src/bootstrap/test.rs

```diff
@@ -971,7 +971,7 @@ impl Step for Compiletest {
 
             // Only pass correct values for these flags for the `run-make` suite as it
             // requires that a C++ compiler was configured which isn't always the case.
-            if !builder.config.dry_run && suite == "run-make-fulldeps" {
+            if !builder.config.dry_run && mode == "run-make" {
```

Comment:
This change here makes it so that run-make needs to only be run on `--host` targets; we don't build the compiler (or LLVM) for `--target` targets. Could you change `default_test!(RunMake` above to `host_test!`? That should fix the failure.

Reply:
Thanks, @Mark-Simulacrum!